### ISSUE TITLE
vagrant or puppet or docker environment

### ISSUE THREAD

**Issue by joejoseph00** [wet-boew/wet-boew-drupal#1914]
I seem to recall links at the wet-boew-drupal page for a vm environment to run the behat tests and build environment for the wet-boew-drupal distro.  I no longer see this in the welcome page . https://github.com/wet-boew/wet-boew-drupal 

The reason I ask, is we're running the latest 4.x and need behat, I've got the goutte driver working but want to run the selenium2 driver for javascript and need an environment ready with all the prerequisites, I know this has been done before so I was hoping to take advantage of the great work that was already done.

we've got some advanced search interfaces and need to test results from ajax calls and to do that the driver and tests must support ajax.  I'm not sure which versions of the stack will work best so I thought I'd refer to what wet-boew-drupal was using.

Or, are you using the aquia drupal vm for this?
they have a vm as well.

**Comment by joejoseph00** [comment]
I did find the wxt docker environment here:
https://hub.docker.com/r/drupalwxt/site-wxt/

, going to look for the wet-boew one.

**Comment by joejoseph00** [comment]
I am reviewing the aquia drupal vm to see if it will answer some of the questions I have

**Comment by joejoseph00** [comment]
ended up using the selenium docker images..  I'll try to share some helpful documentation soon.
